src/dbg/breakpoint.cpp

```diff
@@ -97,7 +97,8 @@ bool BpNew(duint Address, bool Enable, bool Singleshot, short OldBytes, BP_TYPE
 
 bool BpGet(duint Address, BP_TYPE Type, const char* Name, BREAKPOINT* Bp)
 {
-    ASSERT_DEBUGGING("Export call");
+    if(!DbgIsDebugging())

```

Comment:
Why is this here? Is BpGet called when not debugging? The assert should take care of that (although I never run in debug mode).